### ISSUE TITLE
Поясните как воспользоваться методом FNReadFiscalBarcode

### ISSUE THREAD

**Issue by bahekmel** [shtrih-m/fr_drv_ng#268]
Добрый день.
В Тесте Драйвера ККТ есть кнопка "прочитать QR чека" к данной кнопке привязан метод FNReadFiscalBarcode. В методах библиотеки не могу найти FNReadFiscalBarcode. Как получить QR чека?

**Comment by bahekmel** [comment]
![Снимок экрана от 2023-08-23 09-37-42](https://github.com/shtrih-m/fr_drv_ng/assets/93518985/125dc968-70f8-4f60-aaf6-4a531115d3fc)

**Comment by swex** [comment]
Здравствуйте. Метод ищет документ по DocumentNumber и если этот документ чек формирует строку для как в QR в чеке. В classic такого метода сейчас  нет. Добавим.

**Comment by bahekmel** [comment]
Спасибо. Подскажите, пожалуйста, можете ещё добавить в библиотеку classic метод PrintStringWithWrap. За ранее спасибо.

**Comment by swex** [comment]
Да, актуализируем методы, добавим новые, появившиеся в "Драйвер ФР".

**Comment by swex** [comment]
в актуальном релизе:
      > добавлены методы FNGetImplementation, FNGetOSUSupportStatus, FNGetDocumentSize, PrintStringWithWrap, FNReadFiscalBarcode
